ext/native.go

```diff
@@ -325,7 +325,16 @@ func (tp *nativeTypeProvider) NativeToValue(val any) ref.Val {
 	// This isn't quite right if you're also supporting proto,
 	// but maybe an acceptable limitation.
 	switch refVal.Kind() {
-	case reflect.Array, reflect.Slice:
+	case reflect.Array:
+		refElem := refVal.Type().Elem()
+		if refElem == reflect.TypeOf(byte(0)) {
+			tmp := reflect.New(refVal.Type())
```

Comment:
I am not sure about this part of the code. I think this will create a copy of the byte array, but I did not find any other way of retrieving a byte slice, as the passed in values are not addressable. 

If anyone knows a better way of doing this, I would love to hear it.

Reply:
Slice of bytes can be created using usafe without a copy. 
```go
var arr = [3]byte{1, 2, 3}
var arrSlice []byte = unsafe.Slice((*byte)(unsafe.Pointer(&arr[0])), len(arr))
```

Reply:
I sadly failed to do something like this with reflection. Apparently the reason for arrays being passed via interfaces are not addressable was a design decision to prevent subtle bugs, as arrays will still be passed by copy. 

In this case, this causes the not so nice side effect, that now we need to copy the array a second time (at least this is how i understand it, I might be wrong, but would be highly surprised).